src/main/java/edu/nitt/delta/bustracker/service/DriverService.java

```diff
@@ -0,0 +1,33 @@
+package edu.nitt.delta.bustracker.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
+import edu.nitt.delta.bustracker.model.Driver;
+import edu.nitt.delta.bustracker.repository.DriverRepository;
+import lombok.AllArgsConstructor;
+
+@Service
+@AllArgsConstructor
+public class DriverService {
+
+    private final DriverRepository driverRepository;
```

Comment:
```suggestion
    @Autowired private DriverRepository driverRepository;
```

Change it in other service classes as well